src/shared/components/button/large-button/large-button.css.ts

```diff
@@ -0,0 +1,55 @@
+import { recipe } from "@vanilla-extract/recipes";
+import { typographyVars } from "@/shared/styles/typography.css";
+import { color } from "@/shared/styles/tokens/color.css";
+
+export const buttonStyles = recipe({
+  base: {
+    display: "inline-flex",
+    alignItems: "center",
+    justifyContent: "center",
+    userSelect: "none",
+    whiteSpace: "nowrap",
+    borderRadius: "4px",
+    transition: "all 0.2s ease",
+    width: "34.3rem",
+    padding: "1.2rem 1rem",
+    gap: "0.4rem",
+  },
+
+  variants: {
+    variant: {
+      default: {
+        ...typographyVars.body1,
+        color: color.white[100],
+        backgroundColor: color.black[200],
+        border: "1px solid transparent",
+        // default에는 border가 없고, active에는 border가 있으면
+        // 상태 변경 과정에서 버튼 layout이 2px 밀리며 layout shift 발생
+        // -> transparent border 추가로 layout shift 방지
+      },
+      active: {
+        ...typographyVars.body2,
+        color: color.gray[100],
+        backgroundColor: color.white[100],
+        border: `1px solid ${color.gray[200]}`,
+      },
```

Comment:
```suggestion
[LARGE_BUTTON_VARIANTS.DEFAULT] : {
      /* 스타일 속성 */ 
 },
[LARGE_BUTTON_VARIANTS.ACTIVE] : {
      /* 스타일 속성 */ 
 },
```

버튼 스타일 속성에 대해 `button.ts` 상수로 분리해주신 만큼, **css** 파일 내부 속성에서도 `LARGE_BUTTON_VARIANTS` 상수로 지정해주는 것이 어떨까요? 이렇게 하면 모든 로직이 동일한 상수를 참조해서 `variants` 값을 변경해도 상수 파일만 수정하면 되고, 혹시 모를 오타로 인한 스타일 미적용 문제를 방지할 수 있을 것 같아 이 방식도 한 번 제안드립니다 !!

Reply:
바로 적용했어요!!

Reply:
엄청 빠르다 ...!!!

Reply:
저도 리뷰 달고있었거든요...ㅋㅋㅋ 이제 머지하겠습니다~